website/client/components/guilds/discovery.vue

```diff
@@ -2,10 +2,11 @@
 .row
   sidebar(@search="updateSearch", @filter="updateFilters")
 
-  .col-10
+  .col-10.standard-page
     .clearfix
         h1.page-header.float-left(v-once) {{ $t('publicGuilds') }}
-        b-form-select.float-right.sort-select(v-model='sort', :options='sortOptions')
+        b-dropdown.float-right.sort-select(:text="$t('sort')", right=true)
```

Comment:
I think this should have the sort label left to the dropdown and the selected option shown as the shown text per the last design we have (see the equipment)